lib/tether/tether.go

```diff
@@ -252,6 +258,46 @@ func (t *tether) setMounts() error {
 			return fmt.Errorf("unsupported volume mount type for %s: %s", k, v.Source.Scheme)
 		}
 	}
+	return t.populateVolumes()
+}
+
+func (t *tether) populateVolumes() error {
+	defer trace.End(trace.Begin(fmt.Sprintf("populateVolumes")))
+	// skip if no mounts present
+	if len(t.config.Mounts) == 0 {
+		return nil
+	}
+	// skip if this was done before
+	if _, err := os.Stat(mountsCopied); err == nil {
```

Comment:
Done. Removed check for mountsCopied 

Added functionality, if the mounted volume is non empty (except for lost+found) skip the copy flow.